test/DurableTask.Sdk.Tests/PageableIntegrationTests.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using FluentAssertions;
+using Microsoft.DurableTask.Grpc;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DurableTask.Tests;
+
+public class PageableIntegrationTests : IntegrationTestBase
```

Comment:
I was expecting that we'd have an integration test that uses your new `DurableTaskClient.GetInstances(OrchestrationQuery?)` API to query against created orchestration instances. Is that coming for this PR?

Reply:
Oh okay. Yeah let me add that.

Reply:
They are added. Also added some `GetInstanceMetadata` tests. Hopefully these tests show the purpose of `AsyncPageable<T>`. One of the main things hit upon in that test is how pageable can be used to capture a query and then pause/resume it at different points via continuation tokens.